akka-stream/src/main/scala/akka/stream/impl/Stages.scala

```diff
@@ -126,6 +125,7 @@ private[stream] object Stages {
       Fused(ops, name(ops.iterator.map(x ⇒ Logging.simpleName(x).toLowerCase).mkString("+")))
   }
 
+  //FIMXE this is not used.
```

Comment:
You can just remove this class instead then. It will be very obvious when taking the appropriate ticket to add the required stage. It might look completely different anyway.